src/compute-client/src/controller/instance.rs

```diff
@@ -2663,7 +2636,7 @@ impl<T: ComputeControllerTimestamp> ReplicaState<T> {
         let map = serde_json::Map::from_iter([
             field("id", id.to_string())?,
             field("collections", collections)?,
-            field("failed", failed)?,
+            field("failed", client.is_failed())?,
```

Comment:
We usually don't log derived fields in `dump`. Seems fine to just drop this one.